packages/frontend/cypress/integration/specs/01-trade-long.spec.js

```diff
@@ -0,0 +1,345 @@
+/// <reference types="cypress" />
+
+import TradePage from '../pages/trade'
+import BigNumber from 'bignumber.js'
+
+const trade = new TradePage()
+
+describe('Trade on trade page', () => {
+  context('Before tests', () => {
+    it(`Before tests`, () => {
+      cy.disconnectMetamaskWalletFromAllDapps()
+      cy.visit('/')
+    })
+  })
+  context('Connect metamask wallet', () => {
+    it(`should login with success`, () => {
+      trade.connectBrowserWallet()
+      trade.acceptMetamaskAccessRequest()
+      cy.get('#wallet-address').should(`contain.text`, '0x' || '.eth')
+    })
+  })
+  context('Trade switch checks', () => {
+    it('switch between long & short, open & close trade cards with default 0 value', () => {
+      cy.get('#long-card-btn').click({ force: true })
```

Comment:
What if we check if the open tab is open by default when you switch between Long and Short? Since it works that way at the moment.